kovri.md

```diff
@@ -1,64 +1,68 @@
+RingCT hides the amount of Monero that's transmitted.
+However, some personally identifiable information will still be leaked in the process of sending Monero.
+This video focuses on the technology that closes this gap of privacy. This technology is called Kovri.
 
 Suppose Alice wants to send a transaction to Bob.
 She generates a transaction on her local device, and gets ready to broadcast it to the network, so it can be included in the next block.
-Alice is connected to only a small portion of the nodes on the Monero network. Her transaction request will need to reach as many nodes as possible - ideally all of them - for her to have the greatest chance of this transaction getting included in the next block.
-Let's say she is connected to five other nodes, so she broadcasts her transaction to all five.
-These five nodes then forward her transaction to all the nodes they are connected to, and so on, until many (or all) nodes have this transaction.
+Alice is connected to only a small portion of the nodes on the Monero network. Her transaction request will need to reach as many nodes as possible - ideally all of them - for her to have the greatest chance of this transaction to be included in the next block.
+Let's say she connects to five other nodes. We'll call each node "Bob". Once connected, she broadcasts her transaction to all five Bobs.
+These Bobs then forward her transaction to all the nodes they are connected to, and so on, until many (or all) nodes have this transaction.
 
-Unfortunately, Alice takes a risk to broadcast her transaction.
-Some of the nodes on the network may be logging the IP addresses that they get transaction requests from.
+Unfortunately, Alice takes a risk when broadcasting her transaction.
+Some of the nodes on the network may be logging the IP addresses they get when transaction requests are given.
 An IP address is like your home address for your internet connection.
 Attackers will try to associate transactions with the IP addresses they originate from.
 They do not know anything about what address a transaction is coming from, where it is going to, or what the amount is.
 
-However, an attacker with enough resources could knock on Alice's door and demand that she tell them this information. 
+However, an attacker with enough resources could knock on Alice's door and demand that she tell them this information.
 Alternatively, they could purposefully attempt to block her transaction from being broadcast to the rest of the network.
 
 So what can Alice do to mitigate these threats?
 
-Alice does not know which nodes log her IP address, so she can't simply avoid the malicious nodes.
-Her only choice is to hide her IP address.
-There are several ways to hide an IP address, including with Tor, a trusted VPN, or I2P.
-We will focus on I2P in today's video, since this is the network on which Kovri operates.
+Alice does not know which Bob logs her IP address, so she can't simply avoid the malicious Bobs.
```

Comment:
I suggest removing lines 27-39. It's good content, but perhaps is not best for this video.

Reply:
Agreed. I can put it in the kovri FAQ or similar.